tests/crypto/random_tests.c

```diff
@@ -13,38 +13,48 @@
 #include "tests.h"
 
 #define SEQ_COUNT 64
-#define SEQ_LENGTH 19
 
-void TestRandom(void)
+static void _test_random(size_t seq_length)
 {
-    printf("=== begin %s()\n", __FUNCTION__);
+    printf("=== begin %s(%zu)\n", __FUNCTION__, seq_length);
 
-    uint8_t buf[SEQ_COUNT][SEQ_LENGTH];
+    uint8_t buf[SEQ_COUNT][seq_length];
```

Comment:
Ah, didn't catch this the first time. We do not allow variable length arrays. You'll need to allocate `buf` on the heap or make buf sufficiently large for all calls to _test_random.

If you go for the second option, please create a macro with the max value of seq_length.